package.yaml

```diff
@@ -15,11 +15,10 @@ extra-source-files:
 
 dependencies:
 - base >=4.8 && <5
-- async
 - bytestring
 - process >=1.2
 - stm
-- transformers
+- unliftio
```

Comment:
I'd rather depend on `unliftio-core` instead to reduce dependencies, though I realize that will make things a bit more complicated in the usage of `bracket`.